src/views/v2/components/loader/loader.php

```diff
@@ -12,8 +12,8 @@
  * @version TBD
  */
 
-if ( empty( $loader_classes ) ) {
-	$loader_classes = $this->get( 'classes' ) ?: [];
+if ( empty( $classes ) ) {
+	$classes = $this->get( 'classes' ) ?: [];
```

Comment:
:no_entry_sign: Using short ternaries is not allowed

<sub>hash: 3372cb0675ca8a1c75ac094e334efe45</sub>